utils/spdx/src/main/resources/simple-license-mapping.yml

```diff
@@ -23,41 +23,22 @@
 ---
 
 AFLv2.1: AFL-2.1
-ALv2: Apache-2.0
-APACHEv2: Apache-2.0
-APL2: Apache-2.0
 APLv2.0: Apache-2.0
-Apache-2: Apache-2.0
 Apache2.0: Apache-2.0
-Apache2: Apache-2.0
 BSD-3: BSD-3-Clause
 BSD2: BSD-2-Clause
 BSD3: BSD-3-Clause
 Bouncy: MIT
 EDL-1.0: BSD-3-Clause
 FreeBSD: BSD-2-Clause-Views
-GPL-2: GPL-2.0-only
-GPL2: GPL-2.0-only
```

Comment:
I prefer not to remove these entries, but instead separate them.
So, their use can be configured.

Reply:
Making simple license mappings configurable is certainly a good idea, but out of scope of this PR. I first like to get the baseline clean.

Reply:
Agree with @fviernau we can't remove these declared license mappings - this will cause major service disruption for many ORT users.

Reply:
I'm fine with simply dropping this last, stricter commit from the PR. Would that resolve any of your concerns?

Reply:
I would propose a splitting out the (not so risky) ambiguous ones and introduce a `ambiguousMappingsEnabled` configuration option.

Reply:
Please, not yet more configuration options 🙄

Reply:
I'm dropped top-most commit which removed those entries.